src/test/java/hw2/ex2/DifElementsPageTest.java

```diff
@@ -0,0 +1,92 @@
+    public void MyTest_02()  {
+
+        driver.manage().window().maximize();
+
+        //1. Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //2. Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //3. Perform login
+        WebDriverWait waitcicon = new WebDriverWait(driver, 10);
+        waitcicon.until(ExpectedConditions.presenceOfElementLocated(By.id("user-icon")));
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        //4. Assert Username is logged
+        WebDriverWait waitname = new WebDriverWait(driver, 20);
+        waitname.until(ExpectedConditions.presenceOfElementLocated(By.id("user-name")));
+        assertTrue(driver.findElement(By.id("user-name")).isDisplayed());
+        assertEquals(driver.findElement(By.id("user-name")).getText(), "ROMAN IOVLEV");
+
+        //5. Open through the header menu Service -> Different Elements Page
+        driver.findElement(By.cssSelector("ul.uui-navigation.nav.navbar-nav.m-l8>li.dropdown")).click();
+        WebDriverWait list = new WebDriverWait(driver, 10);
+        list.until(ExpectedConditions.presenceOfElementLocated(By.xpath("//a[contains(text(),'Different elements')]")));
+        driver.findElement(By.xpath("//a[contains(text(),'Different elements')]")).click();
+
+        //6. Select checkboxes
+        WebElement checkbox1 = driver.findElement(By.cssSelector("div.checkbox-row>label:nth-child(1)"));
+        WebElement checkbox2 = driver.findElement(By.cssSelector("div.checkbox-row>label:nth-child(3)"));
+        checkbox1.click();
+        checkbox2.click();
+
+        //7. Select radio
+        WebElement radio = driver.findElement(By.cssSelector("div.checkbox-row>label.label-radio:nth-child(4)"));
+        radio.click();
+
+        //8. Select in dropdown
+        Select drpYellow  = new Select(driver.findElement(By.cssSelector("div.colors>select.uui-form-element")));
+        drpYellow.selectByVisibleText("Yellow");
+
+        //9.1 Assert that
+        //•	for each checkbox there is an individual log row and value is corresponded to the status of checkbox
+        assertTrue(driver.findElement(By.xpath("//li[contains(text(),'Water')]")).isDisplayed());
+        assertTrue(driver.findElement(By.xpath("//li[contains(text(),'Wind')]")).isDisplayed());
+
+        //•	for radio button there is a log row and value is corresponded to the status of radio button
+        assertTrue(driver.findElement(By.xpath("//li[contains(text(),'Selen')]")).isDisplayed());
+
+        //•	for dropdown there is a log row and value is corresponded to the selected value
+        assertTrue(driver.findElement(By.xpath("//li[contains(text(),'Yellow')]")).isDisplayed());
```

Comment:
Why don't you move repeated parts to a separate method?

Reply:
move to separate method